configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionZooKeeperClient.java

```diff
@@ -52,6 +52,7 @@
     private static final String DOCKER_IMAGE_REPOSITORY_PATH = "dockerImageRepository";
     private static final String ATHENZ_DOMAIN = "athenzDomain";
     private static final String QUOTA_PATH = "quota";
+    private static final String DEDICATED_CLUSTER_CONTROLLER_CLUSTER_PATH = "dedicatedClusterControllerCluster";
```

Comment:
Consider documenting what this node is for, compared to the one on application path (and perhaps document the other one as well)